dashboard/app/models/survey_result.rb

```diff
@@ -18,6 +18,8 @@
 class SurveyResult < ActiveRecord::Base
   include SerializedProperties
 
+  SYSTEM_DELETED = 'system_deleted'.freeze
```

Comment:
Yep. It seems useful to delineate these responses from responses that were left blank by the user.